### PR TITLE
Unit test fixes

### PR DESCRIPTION
**Pull request by josephshieh**
Made some unit test fixes that were mostly in chronos.tests and schedule.tests

### CODE REVIEW

File: chronos/views.py [hunk 1]
```diff
@@ -51,6 +51,8 @@ def get_shifts(year, month, day=None, user=None, week=None, payperiod=None):
         for shift in shifts:
             shift_date = shift.intime
             this_week = shift_date.isocalendar()[1]
+            if this_week == 52 and int(month) == 1:
+                this_week = 1
```
Comment (swansong) [inline chronos/views.py:57]: Was this block that thing we did to fix how isocalendar handles the first week?

File: chronos/tests.py [hunk 2]
```diff
@@ -108,5 +111,5 @@ def test_get_correct_shifts(self):
         week = 1
         payperiod = 1
 
-        self.assertIsNotNone(c_views.get_shifts(self.request, year, month, day,user,week,payperiod))
-        self.assertIsNotNone(c_views.get_shifts(self.request, year, month, None, None, None, None))
```
Comment (swansong) [inline chronos/tests.py:111]: What purpose did self.request serve?

File: labgeeksrpg_config/templatetags/labgeeksrpg_extras.py [hunk 3]
```diff
@@ -25,10 +25,4 @@ def active_tab(request, pattern):
 def active_css(request):
     """ This template tag is used for determining which css file to use for the site.
     """
-    try:
-        profile = UserProfile.objects.get(user=request.user)
-        if profile.site_skin:
-            return profile.site_skin
-        return 'main.css'
-    except:
-        return 'main.css'
```
Comment (swansong) [inline labgeeksrpg_config/templatetags/labgeeksrpg_extras.py:36]: Does this mean that the psuedo-google skin won't work anymore?

File: chronos/views.py (hunk 1 above)
Comment (abztrakt) [inline chronos/views.py:57]: I'm pretty sure that's what this was.

## 

   Craig M. Stimmel
   Software Engineer, Academic & Collaborative Applications
   UW Information Technology
   University of Washington
   (206) 616-6597

File: labgeeksrpg_config/templatetags/labgeeksrpg_extras.py (hunk 3 above)
Comment (abztrakt) [inline labgeeksrpg_config/templatetags/labgeeksrpg_extras.py:36]: That was the decision we made to simplify mobilizing the skin - for the 
time being, all new skin changes go into main.css.

**Pull request merged**